applications/matter_bridge/Kconfig

```diff
@@ -36,6 +36,46 @@ choice BRIDGED_DEVICE_IMPLEMENTATION
 
 endchoice
 
+if BRIDGED_DEVICE_SIMULATED
+
+choice BRIDGED_DEVICE_SIMULATED_SENSOR_IMPLEMENTATION
+	prompt "Simulated bridged sensor device implementation"
+	default BRIDGED_DEVICE_SIMULATED_SENSOR_AUTOMATIC
+
+	config BRIDGED_DEVICE_SIMULATED_SENSOR_AUTOMATIC
+		bool "Automatically simulated bridged sensor device"
+		help
+		  Enables using simulated bridged sensor devices that are
+		  automatically producing random measurements.
+
+	config BRIDGED_DEVICE_SIMULATED_SENSOR_SHELL
+		bool "Shell controlled simulated bridged sensor device"
+		help
+		  Enables using simulated bridged sensor devices that are
+		  controlled by shell commands.
+
+endchoice
+
+choice BRIDGED_DEVICE_SIMULATED_ONOFF_IMPLEMENTATION
+	prompt "Simulated bridged onoff device implementation"
+	default BRIDGED_DEVICE_SIMULATED_ONOFF_SHELL
+
+	config BRIDGED_DEVICE_SIMULATED_ONOFF_AUTOMATIC
```

Comment:
I'm ok with creating two alternative implementations, so I will leave the decision to you, but I think it would be still ok to just remove simulating on/off state and leave command line option, as the only one available. The random state changes simulation turned out to be a bad idea in practice, as actually we can't predict what should be the expected value. I don't see much value for the user in creating multiple alternative versions of our application and on our side it means another ifdefs in the code.